system-addon/test/functional/mochitest/browser_highlights_section.js

```diff
@@ -0,0 +1,65 @@
+"use strict";
+
+add_task(async function setup() {
+  registerCleanupFunction(async () => {
+    await setUpActivityStreamTest();
```

Comment:
Using a setup function as a cleanup function is a bit dangerous because if someone adds stuff that you wouldn't want to stick around (like changing prefs etc.), it isn't immediately obvious that it would cause problems.

Reply:
This doesn't actually add anything only clears history/bookmarks